chains/v22/chains_dev.json

```diff
@@ -881,20 +881,6 @@
                     "transfersEnabled": true
                 }
             },
-            {
-                "assetId": 19,
-                "symbol": "TEER",
-                "precision": 12,
-                "priceId": "integritee",
-                "type": "orml",
-                "icon": "TEER.svg",
-                "typeExtras": {
-                    "currencyIdScale": "0x050800",
-                    "currencyIdType": "acala_primitives.currency.CurrencyId",
-                    "existentialDeposit": "100000000000",
-                    "transfersEnabled": true
-                }
-            },
```

Comment:
despite that network has gone, token may be available on particular network:

<img width="1248" height="771" alt="Image" src="https://github.com/user-attachments/assets/8a7aa4a0-0874-4791-b095-36db22b3b797" />